src/get-started/quickstart-py.md

```diff
@@ -68,6 +68,24 @@ Install the latest Python SDK from PyPI:
 pip install apache-beam
 ```
 
+#### Extra Requirements
```

Comment:
hmm I thought the line below should address that.

@hadarhg do you have thoughts on how to convey that the user might need to install extra packages highlighted below if they need to use certain features ?